tests/e2e/yamcs/limits.e2e.spec.js

```diff
@@ -197,6 +199,92 @@ test.describe("Mdb runtime limits tests @yamcs", () => {
         // Ensure that the changed limits are now displayed without a reload
         await assertLimitLinesExistAndAreVisible(page);
     });
+
+    test('No new requests for data are made when limits are redrawn due to window resizing', async({ page }) => {
+        // Create an Overlay Plot
+        const overlayPlot = await createDomainObjectWithDefaults(page, {
+            type: 'Overlay Plot'
+        });
+
+        //Expand the myproject folder (/myproject)
+        const myProjectTreeItem = page.locator('.c-tree__item').filter({ hasText: 'myproject'});
+        const firstMyProjectTriangle = myProjectTreeItem.first().locator('span.c-disclosure-triangle');
+        await firstMyProjectTriangle.click();
+
+        //Expand the myproject under the previous folder (/myproject/myproject)
+        const viperRoverTreeItem = page.locator('.c-tree__item').filter({ hasText: 'myproject'});
+        const viperRoverProjectTriangle = viperRoverTreeItem.nth(1).locator('span.c-disclosure-triangle');
+        await viperRoverProjectTriangle.click();
+
+        //Find the Detector_Temp parameter (/myproject/myproject/Detector_Temp)
+        const detectorTreeItem = page.getByRole('treeitem', { name: /Detector_Temp/ });
+
+        // Enter edit mode for the overlay plot
+        await page.click('button[title="Edit"]');
+
+        //Drag and drop the Detector_Temp telemetry endpoint into this overlay plot
+        const objectPane = page.locator('.c-object-view');
+        await detectorTreeItem.dragTo(objectPane);
+
+        // Save (exit edit mode)
+        await page.locator('button[title="Save"]').click();
+        await page.locator('li[title="Save and Finish Editing"]').click();
+
+        // Assert that no limit lines are shown by default
+        await page.waitForSelector('.js-limit-area', { state: 'attached' });
+        expect(await page.locator('.c-plot-limit-line').count()).toBe(0);
+
+        // Enter edit mode
+        await page.click('button[title="Edit"]');
+
+        // Expand the "Detector_Temp" plot series options and enable limit lines
+        await page.getByRole('tab', { name: 'Config' }).click();
+        await page
+            .getByRole('list', { name: 'Plot Series Properties' })
+            .locator('span')
+            .first()
+            .click();
+        await page
+            .getByRole('list', { name: 'Plot Series Properties' })
+            .locator('[title="Display limit lines"]~div input')
```

Comment:
@ozyx what was the workaround to get away from the div on this one

Reply:
I don't have an example but I think we can just chain another `.getByRole()` like so:

```js
await page.getByRole('list', { name: 'Plot Series Properties' })
    .getByTitle('Display limit lines')
    .getByRole('input')
```

I _think_ this works. Either way, it's worth a shot to move away from the CSS selector

Reply:
Done.